sklearn/cluster/_spectral.py

```diff
@@ -292,6 +292,12 @@ def spectral_clustering(
         .. versionchanged:: 1.1
            Added new labeling method 'cluster_qr'.
 
+        For a more extended example, see:
```

Comment:
Could you add the link at the end of the paragraph instead ([here](https://github.com/scikit-learn/scikit-learn/blob/ddc170d41/sklearn/cluster/_spectral.py#L292)) and shorten the description to something like, "For a comparison between the different strategies, see ..."

Reply:
@marenwestermann  This is how I plan to modify it: 

For a detailed comparison of clustering strategies, refer to:
 :ref: `sphx_glr_auto_examples_cluster_plot_coin_segmentation.py`.

 This example demonstrates spectral clustering on an image to segment it into regions based on voxel-to-voxel differences.


Should I delete/keep or make this part shorter: 
"This example demonstrates spectral clustering on an image to segment it into regions based on voxel-to-voxel differences."

Thanks again for keeping an eye on my PR :rocket:

Reply:
Hi @miguelcsx! That sounds good. It's best to delete the second part (the part that starts with "This example demonstrates...") in order to keep the documentation succinct. So only keep the part that starts with, "For a detailed comparison ..."

Reply:
Hello again @marenwestermann , the changes were made :+1: